pkg/webhook/validate_auth.go

```diff
@@ -172,3 +175,12 @@ func contains(arr []string, target string) bool {
 	}
 	return false
 }
+
+func convertExtra(in map[string]authnv1.ExtraValue) map[string]authzv1.ExtraValue {
+	// map from authentication and authorization types
+	extra := map[string]authzv1.ExtraValue{}
```

Comment:
```suggestion
	extra := make(map[string]authzv1.ExtraValue{}, len(in))
```
This might be a little more performant?